### ISSUE TITLE
Galaticraft Launch Button Not working

### ISSUE THREAD

**Issue by Custardstar** [xJon/The-1.12.2-Pack#915]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
yes
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
yes
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
yes

**Information required**
1. Is the modpack crashing:  no
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): **[crash-2021-12-10_04.28.11-client.txt](https://github.com/xJon/The-1.12.2-Pack/files/12754595/crash-2021-12-10_04.28.11-client.txt)** https://mclo.gs/vibDTyA
3. Is Optifine installed or any other additional mods: **yes I am using the 1.12.2 pack on the technic launcher**

**Describe the issue**
Hello,

Me and my gf are playing on a LAN server on the 1.12.2 pack using technic launcher to run it. All has been good so far with the occational crash when we are both doing something that takes alot to run (I upped my RAM to 12gb and it reduced the crashing). Anyways the issue we are currently having is we cant get to any of the planets in the Galaticraft mod. I have used this mod loads before as im a minecraft veteran but ive never had this issue even on LAN, first of all we have tried to  use the planet progression mod to research the moon however it says that the moon and overworld have already been researched even tho in the menu for the planets it doesnt come up with much info about them and when we launch into space using either teir 1 or 2 rockets we cannot press the launch button as its still red on any of the planets exept the overworld. I then tried making a space station which worked well but my gf cant see the space station on her menu and i cant launch to any other worlds other than the overworld from it. Any help on the matter would be very helpful and if you need more info lmk. Hopefully its an easy fix.

Many thanks
Custardstar
A clear and concise descri
[crash-2021-12-10_04.28.11-client.txt](https://github.com/xJon/The-1.12.2-Pack/files/12754595/crash-2021-12-10_04.28.11-client.txt)
ption of what the issue is.

**Comment by xJon** [comment]
Is the crash report you attached the only crash you got? Seems to be some issue relevant the Entity Culling mod, can you try to disable it and try again?

**Comment by Custardstar** [comment]
that was an old crash report mb, my issue isnt that the game crashes i just cant click on the launch button. are there any other files u need?

**Comment by xJon** [comment]
Can you provide a `latest.log` file using https://mclo.gs of the game session in which the launch button doesn't work? Perhaps it printed an error message into the log.
Also, are you using Modular Powersuits? There's a bug with them in space in the current version.

**Comment by Custardstar** [comment]
https://mclo.gs/HaCyRtF is the latest logs.
we arent using modular powersuits but it might be in the modpack

**Comment by xJon** [comment]
For the space station, you can use command `/ssinvite <player>` to invite other players.
Regarding the other moons and planets, are you sure you followed [Planet Progression](https://www.curseforge.com/minecraft/mc-mods/planet-progression) properly? Can you reproduce this issue in a creative test world?

**Comment by Custardstar** [comment]
![image](https://github.com/xJon/The-1.12.2-Pack/assets/146395505/674036cc-7ccb-4ff9-a22b-9036aed52f84)
I have tried playing around with the planet progression mod as i thought there might be an issue with the pack auto placing the moon into my planets that ive progressed (or the mod just auto gives u the moon idk) but i used the command
/unlockAllCelestialBodies
which i got from the galaticraft from https://forum.micdoodle8.com/index.php?threads/1-8-9-1-12-2-planet-progression-updated-12-08-2020.6847/
anyways i did that and it unlocked all the planets so i shot up to space quickly in a spawned in tier 10 creative rocket and i saw this page where some planets were red where i cant go there and some were okay. However... the moon still has a red Launch button so i again am at a stump.
Also i did this test on the world me and my gf use exept i hadnt clicked the open to lan button yet to avoid messing with the data and eliminating the issue being LAN related. I have also tried creating a whole new world and trying that way and it didnt work.

Again many thanks
Custardstar

**Comment by Custardstar** [comment]
we followed the mod correctly before i tried all this and it still wasnt working

**Comment by xJon** [comment]
It's strange, I'm unable to reproduce this issue in a test world. In fact, the Moon doesn't require any progression research, it should be unlocked by default.
I created a new world and got a pre-fueled rocket, put it on launch pads, and went to space, as you can see the launch button for the Moon is green:
![image](https://github.com/xJon/The-1.12.2-Pack/assets/8945347/7f4014f2-580b-44f3-ab41-9b4d51490b95)
 Can you tell me perhaps more in detail how did you reproduce the issue in a new world?

**Comment by Custardstar** [comment]
its not an issue i really created or only happens on odd occations, every time we go on i try and go to the moon the same way as you just did then and it never works or changes. If its unlocked by default for you automatically prehaps its something to do with that?
the first time i attempted to go to the moon me and my gf both built level 1 rockets turned oil into fuel with the galaticraft refiner filled them up using the fuel loaders and took off. We both had the full suits on level 1 that i think we did spawn in and both have a big backpack and some basic stuff for what we needed to make a gas burning generator and a basic base. however when we reached space neither of us could press on the button. We plan on a lan server together.
